repository/spark/operator/params.yaml

```diff
@@ -148,3 +148,39 @@ parameters:
     description: "Install sparkapplications.sparkoperator.k8s.io and scheduledsparkapplications.sparkoperator.k8s.io"
     default: true
     displayName: "Install Spark Operator CRDs"
+
+  - name: replicas
+    description: "The number of replicas of the operator Deployment"
+    default: 1
+    displayName: "Number of replicas"
+
+  - name: enableLeaderElection
+    description: "Whether to enable leader election when the operator Deployment has more than one replica, i.e., when replicas is greater than 1."
+    default: true
```

Comment:
Given that the default number of replicas is "1", does it make sense to set the default to `false` here?

Reply:
Good catch, thank you @nfn! Updated default value to `false`.